proc-macros/src/attributes.rs

```diff
@@ -63,7 +63,7 @@ impl Rpc {
 pub(crate) struct Method {
 	/// Method name
 	pub name: syn::LitStr,
-	/// Alias for the method.
+	/// Aliases for the method.
```

Comment:
Same thing below, lines 77 and 79. And The `alias` member should be in the plural as well.